### PR TITLE
[IE Samples Docs] Add rows with model download and conversion

### PR DESCRIPTION
**Pull request by dpigasin**
### Tickets:
 - *57662*

### CODE REVIEW

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md [hunk 1]
```diff
@@ -47,33 +47,45 @@ To run the sample, you need specify a model and image:
...
 ## Sample Output
 
 The application outputs top-10 inference results.
 
-```sh
+```
 Top 10 results:
 
-Image /opt/intel/openvino/deployment_tools/demo/car.png
+Image C:\images\car.bmp
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/c/samples/hello_classification/README.md:75]: use path from demo folder

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md [hunk 2]
```diff
@@ -47,33 +47,45 @@ To run the sample, you need specify a model and image:
 >
 > - The sample accepts models in ONNX format (\*.onnx) that do not require preprocessing.
 
-You can do inference of an image using a trained AlexNet network on a GPU using the following command:
+### Example
+1. Download a pre-trained model using [Model Downloader](@ref omz_tools_downloader_README):
+```
+python <path_to_omz_tools>/downloader.py --name <model_name>
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/c/samples/hello_classification/README.md:53]: maybe use the real model name?

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md [hunk 3]
```diff
@@ -47,33 +47,45 @@ To run the sample, you need specify a model and image:
...
+```
+
+2. If a model is not IR or ONNX format, it must be converted. You can do this using the special script:
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/c/samples/hello_classification/README.md:56]:
```suggestion
2. If a model is not Inference Engine IR or ONNX format, it must be converted. You can do this using the model converter script:
```

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md (hunk 1 above)
Comment (dpigasin) [reply inference-engine/ie_bridges/c/samples/hello_classification/README.md:75]: It's about the images that can be downloaded from https://storage.openvinotoolkit.org/data/test_data

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md (hunk 3 above)
Comment (dpigasin) [reply inference-engine/ie_bridges/c/samples/hello_classification/README.md]: done

File: inference-engine/ie_bridges/c/samples/hello_classification/README.md (hunk 2 above)
Comment (dpigasin) [reply inference-engine/ie_bridges/c/samples/hello_classification/README.md]: done

**Review by generalova-kate — APPROVED** [review]

**Pull request merged**